packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -158,6 +160,11 @@ class PaginatedDataTable extends StatefulWidget {
   ///
   /// This value is optional and defaults to 56.0 if not specified.
   final double headingRowHeight;
+  
+  /// The color of the heading row.
+  ///
+  /// This value is optional and defaults to theme primary color if not specified.
+  final double headingRowColor;
```

Comment:
Should we just use the same docs from DataTable.headingRowTable?  
`{@macro flutter.material.dataTable.headingRowColor}`